acapi2/resources/applicationlist.py

```diff
@@ -19,8 +19,8 @@ def __init__(self, uri: str,
         self.fetch()
 
     def fetch(self) -> None:
-        apps = super().request(uri=self.uri,
-                               params=self._qry_params).json()
+        apps = self.request(uri=self.uri,
+                            params=self._qry_params).json()
```

Comment:
What's the issue you're encountering with `super()` vs `self`? (as in expected behaviour vs what does actually happen)